aim/cli/convert/processors/tensorboard.py

```diff
@@ -164,50 +168,72 @@ def get_level(current_path):
+                                    err=True
+                                )
+                                unsupported_plugin_noticed = True
                             continue
 
-                    if plugin_name not in supported_plugins:
-                        if not unsupported_plugin_noticed:
-                            click.echo(
-                                'Found unsupported plugin type in the log file. '
-                                'Data for these wont be processed. '
-                                'Supported plugin types are: {}'.format(', '.join(supported_plugins)),
-                                err=True
-                            )
-                            unsupported_plugin_noticed = True
-                        continue
-
-                    if plugin_name == 'images':
-                        tensor = value.tensor.string_val[2:]
-                        track_val = [
-                            Image(tf.image.decode_image(t).numpy()) for t in tensor
-                        ]
-                        if len(track_val) == 1:
-                            track_val = track_val[0]
-                    else:
-                        d_type = tf.dtypes.DType(value.tensor.dtype)
-                        decoded = tf.io.decode_raw(value.tensor.tensor_content, d_type)
-                        track_val = float(decoded)
-
-                    run_tb_log['values'][value_id] = {
-                        'step': step,
-                        'timestamp': timestamp
-                    }
-                    run._track_impl(track_val, timestamp, tag, step, context=event_context)
+                        try:
+                            if plugin_name == 'images':
+                                tensor = value.tensor.string_val[2:]
+                                track_val = [
+                                    Image(tf.image.decode_image(t).numpy()) for t in tensor
+                                ]
+                                if len(track_val) == 1:
+                                    track_val = track_val[0]
+                            else:
+
+                                if value.HasField('simple_value'):
+                                    track_val = value.simple_value
+                                else:
+                                    track_val = value.tensor.float_val[0]
+
+                        except RuntimeError as exc:
+                            # catch all the nasty failures
+                            fail_count += 1
+                            if not _err_info:
+                                _err_info = str(exc)
+                            continue
+
+                        run_tb_log['values'][value_id] = {
+                            'step': step,
+                            'timestamp': timestamp
+                        }
+                        run._tracker._track(track_val, timestamp, tag, step, context=event_context)
```

Comment:
we need to make sure no additional data is tracked automatically. please make sure to pass
`capture_terminal_logs` and `system_tracking_interval` arguments to Run initialization.